src/app/pages/SearchResults.jsx

```diff
@@ -26,6 +26,8 @@ const SearchResults = (props) => {
     sortBy,
   } = props;
 
+  const { includeDrbb } = context;
```

Comment:
Is it possible to use the new context API here?

Reply:
I tried working with the new context API before and found it to be a lot more code. The legacy context has been working fine for our purposes. Plus, that is how we get our RouterContext. For the time being, I'd rather stick to one context API.